kotlinx-coroutines-core/common/src/Builders.common.kt

```diff
@@ -218,13 +218,16 @@ private const val SUSPENDED = 1
 private const val RESUMED = 2
 
 // Used by withContext when context dispatcher changes
-internal class DispatchedCoroutine<in T>(
+@PublishedApi
```

Comment:
Do you mind explicitly writing "Used exclusively by IDEA debugger, see KTIJ-24102" in all such places (both `@PublishedAPI` and actual declarations used)

Reply:
May be it make sense also to introduce internal annotation like `UsedByIDEADebugger` with description instead of just copy pasting the same message?